### DISCUSSION TITLE
How we manage the community?

### DISCUSSION THREAD

**Discussion by binnes** [okd-project/okd#1288] [Ideas]
As we are trying to allow greater participation by our community. outside the RedHat firewall,  we need to think about how we manage activities within the community,  We now have our own github account ([https://github.com/okd-project](https://github.com/okd-project)) and are looking at options to hae build infrastructure outside the RedHat firewall so we need to work out how we want to manage these resources, who will have authority within the different resources, etc.

We also need a way to track activities.  One of the project we have kicked off is to create an OKD operator catalog, so we will need to have processes in place to build and release updates to the operators in the catalog, etc..  Having an activity tracking board will also help new community memebers see where they could contribute.

This seems a good time to have a discussion within the community around the governance and processes we use to manage the OKD community.

Any thoughts / ideas please join the discussions.  These ideas will be brought together for approval of the working group

**Comment by GeoEducator** [comment]
Hello,  I volunteering? ok, I will look up stuff for OKD operator catalog, a testing system...
I remember at the old IBM Network Hardware Division, we would have our set of test cases laid out and would not GA until all were satisfied or work-arounds identified... I assume that is the expectation of the OKD operator catalog for testing... now to make it quick, are their any old process I can read up on and suggest as feasible?
